core/impl-base/src/main/java/org/jboss/arquillian/core/impl/ManagerImpl.java

```diff
@@ -597,6 +599,27 @@ void debug(ObserverMethod method, boolean interceptor)
       }
    }
 
+   private void debugExtension(Class<?> extension)
```

Comment:
The problem is that an extension is a generic class so the name is in that way because if not you read debug(Class<?>) so you don't have any clue that only debug extensions.

Also this method is here because all logging things happens in this class